roles/openshift_facts/library/openshift_facts.py

```diff
@@ -119,24 +119,6 @@ def migrate_node_facts(facts):
     return facts
 
 
-def migrate_hosted_facts(facts):
```

Comment:
I did not re-implement this function.  These should have been migrated long ago.  Do we want to add these to deprecations?